pkg/cmd/pipelinerun/list.go

```diff
@@ -33,30 +33,29 @@ import (
 	cliopts "k8s.io/cli-runtime/pkg/genericclioptions"
 )
 
-const listTemplate = `{{- $prl := len .PipelineRuns.Items }}{{ if eq $prl 0 -}}
+const listTemplate = `{{- $prl := len .PipelineRuns.Items -}}{{- if eq $prl 0 -}}
 No PipelineRuns found
 {{- else -}}
-{{- if $.AllNamespaces }}NAMESPACE	NAME	STARTED	DURATION	STATUS	
-{{- else -}}
+{{- if not $.NoHeaders -}}
+{{- if $.AllNamespaces -}}
```

Comment:
those templates are getting more complex than go codes 🤔 

(it's not an issue with your PR just a thought)

Reply:
I introduced templates in the list command for these two (or one of them can't remmeber exactly) for the --all-namespaces flag, it has made formatting better but in the PR I have refactored the templates to be cleaner and make more sense.